api/openapi-spec/swagger.json

```diff
@@ -173,7 +169,6 @@
    },
    "/apis/kubevirt.io/v1alpha1/namespaces/{namespace}/migrations": {
     "get": {
-     "description": "Get a list of Migration objects.",
```

Comment:
Do we know why these descriptions are getting removed? I wouldn't expect this

Reply:
I asked it under the comment https://github.com/kubevirt/kubevirt/pull/678#issuecomment-361047835, @lukas-bednar can you please check it?

Reply:
@lukas-bednar Hi Lukas, did you find a reason? I think it somehow connects to the updated swagger package.

Reply:
I was trying to reproduce it, but when I rebased I hit issue with dependencies ...

```
[WARN]	Conflict: github.com/emicklei/go-restful rev is currently dc0f94ee75de39d6420e5446b0222490264bb90f, but k8s.io/client-go wants ff4f55a206334ef123e4f79bbf348980da81ca46
[WARN]	Conflict: github.com/mailru/easyjson rev is currently 2a92e673c9a6302dd05c3a691ae1f24aef46457d, but k8s.io/client-go wants 2f5df55504ebc322e4d52d34df6a1f5b503bf26d
[ERROR]	Error scanning k8s.io/apimachinery/pkg/apis/meta/v1alpha1: cannot find package "." in:
	/root/.glide/cache/src/https-k8s.io-apimachinery/pkg/apis/meta/v1alpha1
[ERROR]	Error scanning k8s.io/apimachinery/pkg/conversion/unstructured: cannot find package "." in:
	/root/.glide/cache/src/https-k8s.io-apimachinery/pkg/conversion/unstructured
```

Reply:
@lukas-bednar Sorry for this, I rebased it, can you try again?